File: src/main/java/tech/bts/webserver/data/Product.java
package tech.bts.webserver.data;

public class Product {

    public String name;
    public double price;

    public Product(String name, double price) {
        this.name = name;
        this.price = price;
    }

    public String getName() {
        return name;
    }

    public double getPrice() {
        return price;
    }

    public String toString() {
        return this.name + " - " + this.price + "$";

    }
}

File: src/main/resources/public/product.js
console.log("loaded");

axios.get('/api/products')
    .then(function (response) {
        // handle success
        let products = response.data;
        console.log(products);

        let productsDiv = document.getElementById("products");

        let productsHtml = "";

        for (product of products) {
            productsHtml += "<p>" + product.name + " - " + product.price + "$" + "</p>";
        }

        productsDiv.innerHTML = productsHtml;
    });

